framework/src/modules/chain/submodules/blocks/process.js

```diff
@@ -348,6 +354,7 @@ Process.prototype.getCommonBlock = function(peer, height, cb) {
 			if (comparisonFailed && modules.transport.poorConsensus()) {
 				return modules.blocks.chain.recoverChain(cb);
 			}
+			if (err) err = new Error(err);
```

Comment:
prefer
```
if (err) {
  return setImmediate(cb, new Error(err));
}
```

Reply:
@shuse2 this case is special and needs to stay as it is because err can be null; changing this might require refactoring older methods.

Reply:
yes, im just saying about the format =)